### PR TITLE
2024.2

### PR DESCRIPTION
**Pull request by frenck**
# ⚠️ Please do not merge. This PR is for reviewing purposes.

### CODE REVIEW

File: source/_posts/2024-02-07-release-20242.markdown [hunk 1]
```diff
@@ -0,0 +1,1003 @@
...
+
+<a href='/integrations/#version/2024.2'><img src='/images/blog/2024-02/social.png' style='border: 0;box-shadow: none;'></a>
+
+Home Assistant Core 2024.2! 🥰
+
+Undoubtedly, you've heard about last year's ["The Year of the Voice"](/blog/2022/12/20/year-of-voice/).
+2023 might be over, but we are definitely not done with voice yet! This release
+contains some very cool new features for voice.
+
+Not just that, we've also launched a [voice assistant contest] that you could
```
Comment (edenhaus) [inline source/_posts/2024-02-07-release-20242.markdown]: Link missing for contest

**Review by joostlek — DISMISSED** [review]

File: source/_docs/authentication.markdown [hunk 2]
```diff
@@ -40,6 +40,12 @@ You can:
 - Create [Long Lived Access Tokens](https://developers.home-assistant.io/docs/auth_api/#long-lived-access-token) so scripts can securely interact with Home Assistant.
 - Log out of Home Assistant.
 
+<div class="note">
+
+Unused refresh tokens will be automatically removed. A refresh token is considered unused if it has not been used for a login within 90 days. If you need a permanent token, then we recommend using [Long Lived Access Tokens](/docs/auth_api/#long-lived-access-token).
```
Comment (TheJulianJES) [inline source/_docs/authentication.markdown]: This link doesn't seem to point to the dev site (the one above does).

File: source/_posts/2024-02-07-release-20242.markdown [hunk 3]
```diff
@@ -0,0 +1,1003 @@
...
+We changed the way refresh tokens are handled. Refresh tokens are part of what
+keeps you logged in to Home Assistant.
+
+Refresh tokens will be automatically deleted when unused. A refresh token
+is considered unused if it has not been used for a login within 90 days.
+
+If your logged-in session hasn’t been used for 90 days, you will be logged out.
+You will need to log in again.
+
+We recommend using [Long-lived access tokens](/docs/auth_api/#long-lived-access-token)
```
Comment (TheJulianJES) [inline source/_posts/2024-02-07-release-20242.markdown]: Same here, points to normal/missing docs, not the developer docs.

**Review by janiversen — DISMISSED** [review]
LGTM, thanks

**Review by jbouwh — DISMISSED** [review]
LGTM

File: source/_posts/2024-02-07-release-20242.markdown [hunk 4]
```diff
@@ -0,0 +1,1003 @@
...
+[ZHA](/integrations/zha) (Zigbee Home Automation, our built-in Zigbee integration)
+now provides update entities to Home Assistant!
+
+This means you can get notified when a device has a firmware update available,
+and you can trigger the update from Home Assistant!
+
+<img class="no-shadow" src='/images/blog/2024-02/zha-updating-devices.png' alt='Screenshot showing an Zigbee device connected via ZHA being updated.'>
+
+Noticed the weird version numbers? Unfortunately, that is how version numbers
+in the Zigbee standard work and look. We haven't figured out away to translate
```
Comment (MartinHjelmare) [inline source/_posts/2024-02-07-release-20242.markdown]:
```suggestion
in the Zigbee standard work and look. We haven't figured out a way to translate
```

File: source/_posts/2024-02-07-release-20242.markdown [hunk 5]
```diff
@@ -0,0 +1,1003 @@
...
+automations based on the proximity of people. For example, if you are within
+a certain range from your home and traveling towards it, you could use
+Home Assistant to turn on the lights, open the garage door, turn up the heating,
+and start your favorite playlist in the living room. Welcome home! 🤗
+
+The best part? This integration is now available to set up from the UI!
+Not just that, but it now uses normal sensor entities that we are all already
+familiar with. This will make automating and displaying the data a breeze!
+
+<img class="no-shadow" src='/images/blog/2024-02/proximity.png' alt='Screenshots of an set up proximity integration, the proximity of a person towards the home zone.'>
```
Comment (MartinHjelmare) [inline source/_posts/2024-02-07-release-20242.markdown]:
```suggestion
<img class="no-shadow" src='/images/blog/2024-02/proximity.png' alt='Screenshots of a set up proximity integration, and the proximity of a person moving towards the home zone.'>
```

File: source/_posts/2024-02-07-release-20242.markdown [hunk 6]
```diff
@@ -0,0 +1,1003 @@
...
+  procedures!
+- Using the [utility meter] integration on, for example, your solar inverter,
+  which turns off at night? You can now set the sensor to always be available,
+  even when the source entity is unavailable. Thanks, [@dougiteixeira]!
+- The [TP-Link Smart Home] integration now supports newer devices,
+  including [Tapo-branded] devices! Awesome job [@sdb9696], [@bdraco],
+  and [@rytilahti]!
+- [@AngellusMortis] added new sound event sensors introduced in
+  UniFi Protect version 2.11 to the [UniFi Protect] integration. Nice!
+- Are you an Jinja templating Ninja? [@TNTLarsn] added a new `bitwise_xor`
```
Comment (MartinHjelmare) [inline source/_posts/2024-02-07-release-20242.markdown]:
```suggestion
- Are you a Jinja templating Ninja? [@TNTLarsn] added a new `bitwise_xor`
```

File: source/_posts/2024-02-07-release-20242.markdown [hunk 7]
```diff
@@ -0,0 +1,1003 @@
...
+([@tronikos] - [#105789]) ([documentation](/integrations/google_generative_ai_conversation))
+
+[@tronikos]: https://github.com/tronikos
+[#105789]: https://github.com/home-assistant/core/pull/105789
+
+{% enddetails %}
+
+{% details "HomeWizard Energy" %}
+
+The Gas meter ID sensor is removed andthe meter ID is now shown as a serial
```
Comment (MartinHjelmare) [inline source/_posts/2024-02-07-release-20242.markdown]:
```suggestion
The Gas meter ID sensor is removed and the meter ID is now shown as a serial
```

File: source/_posts/2024-02-07-release-20242.markdown [hunk 8]
```diff
@@ -0,0 +1,1003 @@
...
+([@jbouwh] - [#107274]) ([documentation](/integrations/mqtt))
+
+[@jbouwh]: https://github.com/jbouwh
+[#107274]: https://github.com/home-assistant/core/pull/107274
+
+{% enddetails %}
+
+{% details "Proximity" %}
+
+The proximity entity (`proximit.*`) is deprecated and will be removed in 2024.8.
```
Comment (MartinHjelmare) [inline source/_posts/2024-02-07-release-20242.markdown]:
```suggestion
The proximity entity (`proximity.*`) is deprecated and will be removed in 2024.8.
```

File: source/_posts/2024-02-07-release-20242.markdown [hunk 9]
```diff
@@ -0,0 +1,1003 @@
...
+([@RoboMagus] - [#97208]) ([documentation](/integrations/rest_command))
+
+[@RoboMagus]: https://github.com/RoboMagus
+[#97208]: https://github.com/home-assistant/core/pull/97208
+
+{% enddetails %}
+
+{% details "Roborock" %}
+
+The services `vacuum.start_pause` was deprecated in 2023.8. Support for the
```
Comment (MartinHjelmare) [inline source/_posts/2024-02-07-release-20242.markdown]:
```suggestion
The service `vacuum.start_pause` was deprecated in 2023.8. Support for the
```

File: source/_posts/2024-02-07-release-20242.markdown [hunk 10]
```diff
@@ -0,0 +1,1003 @@
...
+deprecated services has now been completely removed.
```
Comment (MartinHjelmare) [inline source/_posts/2024-02-07-release-20242.markdown]:
```suggestion
deprecated service has now been completely removed.
```

File: source/_posts/2024-02-07-release-20242.markdown [hunk 11]
```diff
@@ -0,0 +1,1003 @@
...
+
+If you are still using them, please adjust your automations and scripts,
```
Comment (MartinHjelmare) [inline source/_posts/2024-02-07-release-20242.markdown]:
```suggestion
If you are still using this service, please adjust your automations and scripts,
```

File: source/_posts/2024-02-07-release-20242.markdown [hunk 12]
```diff
@@ -0,0 +1,1003 @@
...
+use `vacuum.pause` or `vacuum.start` instead.
```
Comment (MartinHjelmare) [inline source/_posts/2024-02-07-release-20242.markdown]:
```suggestion
to use `vacuum.pause` or `vacuum.start` instead.
```

File: source/_posts/2024-02-07-release-20242.markdown [hunk 13]
```diff
@@ -0,0 +1,1003 @@
...
+[#109226]: https://github.com/home-assistant/core/pull/109226
+
+{% enddetails %}
+
+{% details "Tuya" %}
+
+Tuya has provided an easier and improved login method for Home Assistant users.
+
+Having a developer account with Tuya is no longer required; instead, you can
+scan a QR code with your Tuya Smart of Smart Life app to authenticate it
```
Comment (MartinHjelmare) [inline source/_posts/2024-02-07-release-20242.markdown]:
```suggestion
scan a QR code with your Tuya Smart or Smart Life app to authenticate it
```

**Pull request merged**